app/src/main/java/com/example/playlistmaker/ui/medialibrary/fragment/AddPlaylistFragment.kt

```diff
@@ -38,15 +37,24 @@ import com.google.android.material.textfield.TextInputLayout
 import com.markodevcic.peko.PermissionRequester
 import com.markodevcic.peko.PermissionResult
 import kotlinx.coroutines.launch
-import org.koin.androidx.viewmodel.ext.android.viewModel
+import org.koin.androidx.viewmodel.ext.android.getViewModel
+import org.koin.core.parameter.parametersOf
 import java.io.File
 import java.io.FileOutputStream
 
 class AddPlaylistFragment : BindingFragment<FragmentAddPlaylistBinding>() {
 
-    private val viewModel: AddPlaylistFragmentViewModel by viewModel()
+    private var isEdit: Boolean = false
 
-    private lateinit var backDialog: MaterialAlertDialogBuilder
+    private val playlistId: Int by lazy {
```

Comment:
Здесь можно убрать синхронизацию, т.к. доступ к этому полю всегда осуществляется с одного главного потока:
```suggestion
    private val playlistId: Int by lazy(mode = LazyThreadSafetyMode.NONE) {
```